ui/components/multichain/edit-accounts-modal/edit-accounts-modal.test.tsx

```diff
@@ -40,10 +41,21 @@ const render = (
     mockState.metamask.internalAccounts.accounts,
   ) as unknown as MergedInternalAccount[];
 
+  const accountsWithCaipAccountId = accounts.map((account) => {
+    const { namespace, reference } = parseCaipChainId(account.scopes[0]);
```

Comment:
do we need to account for when scopes contain more scopes? also is it necessary to parse the scope into namespace and reference when you could just contact it as is to the address? `${account.scopes[0]}:${account.address}`

Reply:
ehhhhh it's just for fixtures